java-frontend/src/test/java/org/sonar/java/classpath/ClasspathForMainTest.java

```diff
@@ -444,6 +452,10 @@ void should_include_jdk_in_libraries_when_specified_classic() {
     assertThat(elements)
       .hasSize(1)
       .allMatch(file -> file.getName().equals("rt.jar"));
+    assertThat(logTester.logs(LoggerLevel.DEBUG))
+      .hasSize(2)
+      .anyMatch(debug -> debug.startsWith("Property 'sonar.java.jdkHome' resolved with:"))
+      .anyMatch(debug -> debug.contains("rt.jar"));
   }
 
   @Test
```

Comment:
Looks like this test and the above one are pretty similar. Will be nice if you can replace it with one parametrised.